### PR TITLE
Adding zap full scan github action blog

### PR DESCRIPTION
**Pull request by sshniro**

### CODE REVIEW

File: site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: "Dynamic Application Security Testing with OWASP ZAP and GitHub Actions"
```
Comment (psiinon) [inline site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md:2]: Remove the "OWASP" to keep it consistent with the previous blog post?

File: site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+addSocialPreview: true
+authors:
+    - nirojan
+---
+
+[![zap-action](./images/zap-action.png)](https://github.com/marketplace/actions/owasp-zap-full-scan)
+
+[ZAP full scan GitHub action]((https://github.com/marketplace/actions/owasp-zap-full-scan)) provides free dynamic analysis 
+security testing (DAST) to your web applications. DAST is also known as black-box testing, which allows ZAP to identify 
+potential vulnerabilities in your web applications. ZAP previously introduced the [ZAP baseline scan GitHub action](https://github.com/marketplace/actions/owasp-zap-baseline-scan)
```
Comment (psiinon) [inline site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md:19]: "We previously introduced" maybe?

File: site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+[ZAP full scan GitHub action]((https://github.com/marketplace/actions/owasp-zap-full-scan)) provides free dynamic analysis 
+security testing (DAST) to your web applications. DAST is also known as black-box testing, which allows ZAP to identify 
+potential vulnerabilities in your web applications. ZAP previously introduced the [ZAP baseline scan GitHub action](https://github.com/marketplace/actions/owasp-zap-baseline-scan) 
+to passively identify potential alerts in a web application. However, unlike the baseline scan, ZAP full scan attacks the web application 
+to find additional vulnerabilities.
+
+The ZAP full scan action runs the ZAP spider against the specified target (by default with no time limit) followed by an optional ajax spider scan and then a full active scan before reporting the results. The alerts will be maintained as a GitHub issue in the corresponding repository. This means that the script does perform actual ‘attacks’ and can potentially run for a long period of time.
+
+** Warning **
```
Comment (psiinon) [inline site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md:25]: Doesnt show as bold ;)

File: site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+As a full scan will perform attacks against the target you should only use the full scan against targets that you have 
+permission to test. You should also check with your hosting company and any other services such as CDNs that may be 
+affected before running this action. ZAP will also submit forms which could result in a large number of messages via, 
+for example, 'Contact us' or 'comment' forms.
```
Comment (psiinon) [inline site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md:29]: Link to the relevant FAQ?

File: site/content/blog/2020-04-09-automate-security-testing-with-zap-and-github-actions/index.md [hunk 5]
```diff
@@ -110,5 +110,7 @@ Also, you can view the scan logs by navigating to the ZAP [scan job](https://git
 
 
 We are excited to hear your thoughts and feedback for the new ZAP Baseline Action. If you find the action useful please star the action.
-The ZAP team is also working on a ZAP Full Scan Action, stay tuned for updates. Reach us through the [dev group](https://groups.google.com/group/zaproxy-develop) 
-or for any additional enhancements please feel free to create an issue in the [following repository](https://github.com/zaproxy/action-baseline).
```
Comment (psiinon) [inline site/content/blog/2020-04-09-automate-security-testing-with-zap-and-github-actions/index.md:114]: Shouldnt have delected this line?

File: site/content/blog/2020-04-09-automate-security-testing-with-zap-and-github-actions/index.md [hunk 6]
```diff
@@ -110,5 +110,7 @@ Also, you can view the scan logs by navigating to the ZAP [scan job](https://git
...
+Also checkout out our [ZAP full scan action](https://github.com/marketplace/actions/owasp-zap-full-scan) on how to perform active scanning 
+on your web applications. The ZAP team is also working on a ZAP Full Scan Action, stay tuned for updates. Reach us through the
```
Comment (psiinon) [inline site/content/blog/2020-04-09-automate-security-testing-with-zap-and-github-actions/index.md:114]: Should have deleted the "The ZAP team is also.." part?

File: site/content/blog/2020-04-09-automate-security-testing-with-zap-and-github-actions/index.md [hunk 7]
```diff
@@ -110,5 +110,6 @@ Also, you can view the scan logs by navigating to the ZAP [scan job](https://git
 
 
 We are excited to hear your thoughts and feedback for the new ZAP Baseline Action. If you find the action useful please star the action.
-The ZAP team is also working on a ZAP Full Scan Action, stay tuned for updates. Reach us through the [dev group](https://groups.google.com/group/zaproxy-develop) 
-or for any additional enhancements please feel free to create an issue in the [following repository](https://github.com/zaproxy/action-baseline).
+Also, checkout our [ZAP full scan action](https://github.com/marketplace/actions/owasp-zap-full-scan) on how to perform active scanning 
+on your web applications. Reach us through the [dev group](https://groups.google.com/group/zaproxy-develop) or for any additional
```
Comment (kingthorin) [inline site/content/blog/2020-04-09-automate-security-testing-with-zap-and-github-actions/index.md:114]: Shall we change this to User Group?

People won't really be posting about development of ZAP.

File: site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+        with:
+          target: 'https://www.zaproxy.org/'
+```
+
+Also, you can view the scan logs by navigating to the ZAP [scan job](https://github.com/zaproxy/zaproxy-website/runs/629858267?check_suite_focus=true).
+
+[![scan-job](./images/scan-job.png)](https://github.com/zaproxy/zaproxy-website/runs/629858267?check_suite_focus=true)
+
+We are excited to hear your thoughts and feedback for the ZAP full scan action. If you find the action useful please star 
+the action. Reach us through the [dev group](https://groups.google.com/group/zaproxy-develop) or for any additional enhancements
```
Comment (kingthorin) [inline site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md:79]: Shall we change this to User Group?

People won't really be posting about development of ZAP.

File: site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+- blog
+date: "2020-05-15"
+addSocialPreview: true
+authors:
+    - nirojan
+---
+
+[![zap-action](./images/zap-action.png)](https://github.com/marketplace/actions/owasp-zap-full-scan)
+
+[ZAP full scan GitHub action](https://github.com/marketplace/actions/owasp-zap-full-scan) provides free dynamic analysis
```
Comment (kingthorin) [inline site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md:17]:
```suggestion
[ZAP full scan GitHub action](https://github.com/marketplace/actions/owasp-zap-full-scan) provides free dynamic application 
```

File: site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+date: "2020-05-15"
+addSocialPreview: true
+authors:
+    - nirojan
+---
+
+[![zap-action](./images/zap-action.png)](https://github.com/marketplace/actions/owasp-zap-full-scan)
+
+[ZAP full scan GitHub action](https://github.com/marketplace/actions/owasp-zap-full-scan) provides free dynamic analysis 
+security testing (DAST) to your web applications. DAST is also known as black-box testing, which allows ZAP to identify
```
Comment (kingthorin) [inline site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md:18]:
```suggestion
security testing (DAST) of your web applications. DAST is also known as black-box testing, which allows ZAP to identify 
```

File: site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+potential vulnerabilities in your web applications. We previously introduced the [ZAP baseline scan GitHub action](https://github.com/marketplace/actions/owasp-zap-baseline-scan) 
+to passively identify potential alerts in a web application. However, unlike the baseline scan, ZAP full scan attacks the web application 
+to find additional vulnerabilities.
+
+The ZAP full scan action runs the ZAP spider against the specified target (by default with no time limit) followed by an optional ajax spider scan and then a full active scan before reporting the results. The alerts will be maintained as a GitHub issue in the corresponding repository. This means that the script does perform actual ‘attacks’ and can potentially run for a long period of time.
+
+**Warning:** 
+As a full scan will perform attacks against the target you should only use the full scan against targets that you have 
+permission to test. You should also check with your hosting company and any other services such as CDNs that may be 
+affected before running this action. ZAP will also submit forms which could result in a
```
Comment (kingthorin) [inline site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md:28]:
```suggestion
affected before running this action. ZAP will submit forms and exercise functionality which could result in a 
```

File: site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+
+**Warning:** 
+As a full scan will perform attacks against the target you should only use the full scan against targets that you have 
+permission to test. You should also check with your hosting company and any other services such as CDNs that may be 
+affected before running this action. ZAP will also submit forms which could result in a 
+[large number of messages](/faq/how-can-i-prevent-zap-from-sending-me-1000s-of-emails-via-a-contact-us-form/) via, 
+for example, 'Contact us' or 'comment' forms.
+
+The ZAP full scan action can be used in multiple ways. The action can be configured to run against a publicly available 
+web application or against a locally available web application. Here in ZAP we use the action to periodically scan our
```
Comment (kingthorin) [inline site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md:33]:
```suggestion
web application or against a locally available web application. The ZAP team use the action to scan our 
```

File: site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md [hunk 13]
```diff
@@ -0,0 +1,80 @@
...
+regarding the identified alerts. The report is available in HTML and Markdown formats.
+
+The following issue shows how the GitHub full scan action scans the [https://www.zaproxy.org](https://www.zaproxy.org) 
+website and notifies the users via opening an issue in the ZAP website repository (Scan time: 45 Mins). The issue will be created by the 
+GitHub Actions bot and will list the alerts as issue comments. Compared to our [baseline scan results](https://github.com/zaproxy/zaproxy-website/issues/93#issue-597219582) 
+you can see the full scan has identified additional alerts in the zaproxy website.
+
+[![issue open](images/zap-issue-1.png)](https://github.com/zaproxy/zaproxy-website/issues/110#issue-609117077)
+
+In consequent scans if it identifies a fixed issue a newly raised alerts the action will update the issue with the required information.
```
Comment (kingthorin) [inline site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md:46]:
```suggestion
In consequent scans if it identifies a fixed issue or newly raised alerts the action will update the issue with the required information.
```

File: site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md [hunk 14]
```diff
@@ -0,0 +1,80 @@
...
+[![comment with issues resolved](images/zap-issue-2.png)](https://github.com/zaproxy/zaproxy-website/issues/110#issuecomment-626256038)
+
+If you have resolved/ignored all the alerts then, the action will close the ongoing open issue. You can also use the 
+rules file to ignore the false positives which occur during the ZAP scan. Please check out our previous blog on the
```
Comment (kingthorin) [inline site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md:50]:
```suggestion
rules file to ignore false positives which occur during the ZAP scan. Please check out our previous blog on the 
```

File: site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md [hunk 15]
```diff
@@ -0,0 +1,80 @@
...
+you can see the full scan has identified additional alerts in the zaproxy website.
+
+[![issue open](images/zap-issue-1.png)](https://github.com/zaproxy/zaproxy-website/issues/110#issue-609117077)
+
+In consequent scans if it identifies a fixed issue a newly raised alerts the action will update the issue with the required information.
+[![comment with issues resolved](images/zap-issue-2.png)](https://github.com/zaproxy/zaproxy-website/issues/110#issuecomment-626256038)
+
+If you have resolved/ignored all the alerts then, the action will close the ongoing open issue. You can also use the 
+rules file to ignore the false positives which occur during the ZAP scan. Please check out our previous blog on the 
+[baseline scan](/blog/2020-04-09-automate-security-testing-with-zap-and-github-actions/) on how to exclude false positive from your issues.
```
Comment (kingthorin) [inline site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md:51]:
```suggestion
[baseline scan](/blog/2020-04-09-automate-security-testing-with-zap-and-github-actions/) which covers how to exclude false positive from your issues.
```

File: site/content/blog/2020-04-09-automate-security-testing-with-zap-and-github-actions/index.md (hunk 7 above)
Comment (kingthorin) [reply site/content/blog/2020-04-09-automate-security-testing-with-zap-and-github-actions/index.md:114]:
```
[users group](https://groups.google.com/forum/#!forum/zaproxy-users)
```

File: site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md (hunk 8 above)
Comment (kingthorin) [reply site/content/blog/2020-05-15-dynamic-application-security-testing-with-zap-and-github-actions/index.md:79]:
```
[users group](https://groups.google.com/forum/#!forum/zaproxy-users)
```

File: site/content/blog/2020-04-09-automate-security-testing-with-zap-and-github-actions/index.md (hunk 7 above)
Comment (kingthorin) [reply site/content/blog/2020-04-09-automate-security-testing-with-zap-and-github-actions/index.md:114]: This one wasn't changed.

Comment (sshniro) [reply site/content/blog/2020-04-09-automate-security-testing-with-zap-and-github-actions/index.md]: Done

**Pull request merged**